diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/NullChecksRule.kt

```diff
@@ -72,20 +78,77 @@ class NullChecksRule(private val configRules: List<RulesConfig>) : Rule("null-ch
                     ElementType.EQEQ, ElementType.EQEQEQ ->
                         warnAndFixOnNullCheck(condition, true,
                             "use '.let/.also/?:/e.t.c' instead of ${condition.text}") {
-                            // todo implement fixer
+                            fixNullInIfCondition(node, condition.node, true)
                         }
                     // `!==` and `!==` comparison can be fixed with `.let/also` operators
                     ElementType.EXCLEQ, ElementType.EXCLEQEQEQ ->
                         warnAndFixOnNullCheck(condition, true,
                             "use '.let/.also/?:/e.t.c' instead of ${condition.text}") {
-                            // todo implement fixer
+                            fixNullInIfCondition(node, condition.node,false)
                         }
                     else -> return
                 }
             }
         }
     }
 
+    @Suppress("UnsafeCallOnNullableType")
+    private fun fixNullInIfCondition(condition: ASTNode,
+                                     binaryExpression: ASTNode,
+                                     isEqualToNull:Boolean) {
+        val variableName = binaryExpression.firstChildNode.text
+        val thenCodeLines = condition
+                .treeParent
+                .getFirstChildWithType(THEN)!!
+                .text
+                .drop(1)
+                .dropLast(1)
```

Comment:
```suggestion
                .trim('{', '}')
```
Is it for this? I think explicitly stating removed characters will be more readable.

Reply:
Fixed